Source/Extensions/Blazorise.DataGrid/DataGrid.razor.cs

```diff
@@ -3791,5 +3844,15 @@ public int SelectedRowIndex
     /// </summary>
     [Parameter] public bool AutoGenerateColumns { get; set; } = true;
 
+    /// <summary>
+    /// Gets or sets DataGridEditMode options.
+    /// </summary>
+    [Parameter] public DataGridEditModeOptions DataGridEditModeOptions { get; set; }

```

Comment:
I don't like the DataGridEditModeOptions parameter name. Rename it to `EditModeOptions`. 

PS. the parameter, not the type.

Reply:
Don't like it either, an oversight I guess. :)